.cargo/config

```diff
@@ -11,4 +11,4 @@ rustflags = ["-C", "target-feature=+crt-static"]
 
 [unstable]
 bindeps = true
-sparse-registry = true
+# sparse-registry = true
```

Comment:
As already mentioned in https://github.com/enarx/enarx/pull/2054#discussion_r921169839, please document the reasoning for this change in the commit.

The line itself should be either removed completely, or commented-out with an issue reference to reenable it in the future